readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -278,7 +278,9 @@ def generate_rtd_data(self, docs_dir, mkdocs_config):
             'global_analytics_code': (
                 None if self.project.analytics_disabled else settings.GLOBAL_ANALYTICS_CODE
             ),
-            'user_analytics_code': analytics_code,
+            "user_analytics_code": analytics_code,
+            "proxied_static_path": self.project.proxied_static_path,
+            "proxied_api_host": self.project.proxied_api_host,
```

Comment:
We were missing including these for mkdocs, but it's only relevant when using a path different from `_/`, so